components/views/login/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState, FormEvent } from 'react'
+import styles from './Login.module.scss'
+import useForm from '../../../utilities/hooks/useForm'
+import Link from 'next/link'
+
+const Login = () => {
+    const { input, handleChange, handleReset, clearForm } = useForm({
+        email: '',
+        password: ''
+    })
+    const [error, setError] = useState({
+        message: 'Enter message',
+        hasError: false
+    })
+
+    const { email, password } = input
+    const handleSubmit = (event: FormEvent<HTMLFormElement>) => {
+        event.preventDefault()
+        console.log(input)
+
+        /*
```

Comment:
Remove the comment

Reply:
hating on my future plans

Reply:
Commment has been removed only physically . it will live eternally in my mind and will never ( probably will) be forgotten